scripts/private/sparkk8s_token.sh

```diff
@@ -19,69 +19,66 @@ fi
 USERNAME="$1"
 SERVICE_ACCOUNT="spark"
 
-KUBECONFIG="/srv/jupyterhub/private/sparkk8s.cred"
+export KUBECONFIG="/srv/jupyterhub/private/sparkk8s.cred"
```

Comment:
Why do we need to export this variable?

Reply:
indeed exporting KUBECONFIG does not appear to be strictly needed in this script